zip_wheels.sh

```diff
@@ -27,6 +27,7 @@ zip_wheel() {
     rm -f $2
     cd ./tmp_build_zip
     unzip trulens*-py3-none-any.whl
+    rm -rf `find ./ -name snowflake_stage_zips`
```

Comment:
wouldn't this be handled on line 38? Here it would remove all `snowflake_stage_zips` each time we zip a new package

Reply:
this is removing it from the wheel that was built in the first round of `make build`.